src/tinykeys.ts

```diff
@@ -132,24 +130,19 @@ function match(event: KeyboardEvent, press: KeyBindingPress): boolean {
  * 		alert("Either 'Control+d' or 'Meta+d' were pressed")
  * 	},
  * })
+ *
+ * window.addEvenListener("keydown", handler)
  * ```
  */
-export default function keybindings(
-	target: Window | HTMLElement,
-	keyBindingMap: KeyBindingMap,
-	options: KeyBindingOptions = {},
-): () => void {
-	let timeout = options.timeout ?? DEFAULT_TIMEOUT
-	let event = options.event ?? DEFAULT_EVENT
-
+export function createKeybindingsHandler(keyBindingMap: KeyBindingMap, timeout: number = DEFAULT_TIMEOUT): EventListener {
```

Comment:
Could you make this second parameter a `options` object?

```ts
export interface KeyBindingHandlerOptions {
  // ...
  timeout?: number
}
export interface KeyBindingOptions extends KeyBindingsHandlerOptions {
  // ...
  event?: "keydown" | "keyup"
}
```

Reply:
Sure. Good idea. Made the change and squashed into one commit

Reply:
And this time i let prettier do its thing on the ts file, since there was inconsistent use of spaces for indentation